key/keys.go

```diff
@@ -110,21 +122,28 @@ func NewTLSKeyPair(address string) *Pair {
 
 // PairTOML is the TOML-able version of a private key
 type PairTOML struct {
-	Key string
+	Key        string
+	SchemeName string
 }
 
 // PublicTOML is the TOML-able version of a public key
 type PublicTOML struct {
-	Address   string
-	Key       string
-	TLS       bool
-	Signature string
+	Address    string
+	Key        string
+	TLS        bool
+	Signature  string
+	SchemeName string
```

Comment:
TOML To/From functions have been one of my nightmare in this PR. I'll also need to check all of them in an upcoming commit to make sure they support decoding any TOML coming from the "old code" without the scheme name.

Reply:
you could have multiple structs representing the formats, try the new format, and try the old format if it errors